torch/nn/utils/rnn.py

```diff
@@ -158,7 +180,20 @@ def pad_packed_sequence(sequence, batch_first=False, padding_value=0):
 
     if batch_first:
         output = output.transpose(0, 1)
-    return output, lengths
+    return output, Variable(torch.LongTensor(lengths))
+
+
+def _symbolic_pad_packed_sequence(g, input, batch_first=False, padding_value=0.0):
+    return g.op("PadPacked", input.data, input.batch_sizes_variable, outputs=2)
```

Comment:
This isn't a real ONNX op, right? Better say so.

Reply:
well, we could make it a real onnx op (if we didn't want to rely 100% on the optimization pass cancelling them out), but for now ++ I'll add a comment